migration_guides/70.x.x-to-71.0.0-migration-guide.md

```diff
@@ -8,6 +8,157 @@ This document outlines the steps needed to migrate to version 71.0.0 of our Desi
 
 This section will list the changes that will impact more than one component
 
+### **_New sizes introduced for margin and padding utility classes_**
+
+[**Link to PR**](https://github.com/ONSdigital/design-system/pull/3285)
+
+-   **Description of change:** New margin and padding sizes and have been introduced and relevant utility classes have been updated with the new corresponding size
+-   **Reason for change:** As we have changed the size of our `rems` to be `16px` we needed to introduce new sizes to support our updated spacing grid.
+
+-   **Components impacted:**
+
+    -   Any component or layout where you have manually set our margin or padding utility classes.
+
+-   **Migration steps:**
+
+    1. Locate all instances of the utility classes in your codebase.
+    2. Replace old size name with new corresponding size name:
+
+    -   <details>
+        <summary><b>Click for example:</b></summary>
+
+        ```scss
+        //OLD Sizes
+        $sizes: (
+            no: 0,
+            xxs: 0.25rem,
+            xs: 0.5rem,
+            s: 1rem,
+            m: 1.5rem,
+            l: 2rem,
+            xl: 3rem,
+            auto: auto,
+        );
+
+        //NEW Sizes
+        $sizes: (
+            no: 0,
+            4xs: 0.125rem,
+            3xs: 0.25rem,
+            2xs: 0.5rem,
+            xs: 0.75rem,
+            s: 1rem,
+            m: 1.25rem,
+            l: 1.5rem,
+            xl: 2rem,
+            2xl: 2.5rem,
+            3xl: 3rem,
+            4xl: 4rem,
+            5xl: 5rem,
+            auto: auto,
+        );
+        ```
+
+        </details>
+
+### **_Update icon sizes naming standard_**
+
+[**Link to PR**](https://github.com/ONSdigital/design-system/pull/3285)
+
+-   **Description of change:** Icon sizes have been updated with the new naming size standard.
```

Comment:
Think this needs a bit more information like what has been done for the padding above